src/components/ListMenu.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from "react";
+import "../styles/style.css";
+
+class ListMenu extends React.Component {
```

Comment:
This is super small but you can remove React from React.Component since you pulled it out in the import